webapp/components/file_upload.jsx

```diff
@@ -261,6 +261,13 @@ class FileUpload extends React.Component {
                 }
             }
         });
+
+        document.addEventListener('keydown', (e) => {
+            //CTRL+U or CMD+U for file uploads
+            if ((e.ctrlKey || e.metaKey) && e.keyCode === 85) {
```

Comment:
Can you add a constant to KeyCodes in `utils/contants.jsx` for 85 and use that here instead? It would look like `if ((e.ctrlKey || e.metaKey) && e.keyCode === Constants.KeyCodes.U) {` for example